README.md

```diff
@@ -1,76 +1,101 @@
-# HTML5 Media
+# Working with HTML5 Media
+
+## Problem Statement
+The internet is a highly interactive environment. Even blogs where text
+may be the most dominant content, audio and video elements often support,
+or even supplement, the content. In the case of 3rd party media sites such
+as YouTube, SoundCloud, and Vimeo, you are given an all-in-one line of code
+with configurable properties that creates the embeddable content for you--
+but how do you do this manually in your application? Often times, you will be working
+with, and manipulating, media that is being uploaded into or generated by your
+app. HTML has evolved dramatically since the days of the generic `embed` tag.
+How is this done today?
```

Comment:
I don't think this introduction supports the content below.

As I read this:

1. The internet is interactive
2. Even text sites use media
3. Big sites provide embed codes, how do you make embed codes?
4. People will upload media into my app that I may want to use
5. Embed, which i don't know anything about as a beginner, has been superceded

3,4 seem to suggest I need to build an app
5 seems to suggest I'm missing something